mypy/checker.py

```diff
@@ -618,7 +617,7 @@ def _visit_overloaded_func_def(self, defn: OverloadedFuncDef) -> None:
         for fdef in defn.items:
             assert isinstance(fdef, Decorator)
             self.check_func_item(fdef.func, name=fdef.func.name)
-            if fdef.func.is_abstract:
+            if fdef.func.abstract_status:
```

Comment:
Please don't rely on specific constants being falsey. Use explicit `==` instead. Otherwise if someone will change numeric value of the constant this code will subtly break.